### PR TITLE
Release updates

### PR DESCRIPTION
**Pull request by KnVerey**
A few minor tweaks to the release process based on recent experiences.

### CODE REVIEW

File: releasing/README.md [hunk 1]
```diff
@@ -96,50 +78,58 @@ GITHUB_TOKEN=deadbeefdeadbeef
 echo $GITHUB_TOKEN | gh auth login --scopes repo --with-token
 ```
 
+
+Instructions on how to get a new OpenAPI sample can be found in the
```
Comment (natasha41575) [inline releasing/README.md:82]: Shouldn't this be under `Consider fetching new OpenAPI data`? Why move it?

File: releasing/README.md [hunk 2]
```diff
@@ -96,50 +78,58 @@ GITHUB_TOKEN=deadbeefdeadbeef
...
 
 
 ## Release `cmd/config`
 
-```
-cd ../kustomize
-```
-
-Pin to the most recent kyaml.
+#### Pin to the most recent kyaml
```
Comment (natasha41575) [inline releasing/README.md:120]: Should we also add to this section that you have to manually bump two of the transformer kyaml dependencies, perhaps linking to a sample pinning PR?

File: releasing/README.md [hunk 3]
```diff
@@ -308,7 +315,28 @@ refreshMaster
...
+```
+gh pr status  # rinse, repeat
+gh pr merge -m
+```
+
+
+## Publish Official Docker Image
 
 [k8s.io]: https://github.com/kubernetes/k8s.io
 [k8s-staging-kustomize]: https://pantheon.corp.google.com/gcr/images/k8s-staging-kustomize?project=k8s-staging-kustomize
```
Comment (natasha41575) [inline releasing/README.md:342]: Let's change this link to https://console.cloud.google.com/gcr/images/k8s-staging-kustomize?project=k8s-staging-kustomize - `pantheon.corp` is a google-internal link, I believe changing it to `console.cloud` makes it an externally viewable link that you have permissions to view.

File: releasing/README.md (hunk 1 above)
Comment (KnVerey) [reply releasing/README.md:82]: Definitely. I'm not sure how I managed, but it's some sort of copy-paste error 😅

File: releasing/README.md (hunk 2 above)
Comment (KnVerey) [reply releasing/README.md:120]: That's what the two new lines of code below do 😄

**Pull request merged**